.github/workflows/dast.yml

```
@@ -0,0 +1,18 @@
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+
+jobs:
+  "dast-test":
+    runs-on: ubuntu-latest
+    container: python:3.11
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)